lib/services/packagist.rb

```diff
@@ -37,14 +36,13 @@ def domain
 
   def full_domain
     if data['domain'].to_s == ''
-      'http://packagist.org'
+      'https://packagist.org'
     else
       data['domain']
-    end.lstrip.sub(/[\/\s]+$/,'')
+    end.lstrip.sub(/[\/\s]+$/,'').sub(/^http:\/\/packagist.org/, 'https://packagist.org')
```

Comment:
The `/m` flag only changes the meaning of `.` to match newline characters. Otherwise, this is a good way to make sure we don't over match.